src/ascii/chain/setting/private/v6_14.rs

```diff
@@ -0,0 +1,4422 @@
+//! Settings in firmware version 6.14.
```

Comment:
These files are _really_ large. Most are 4000-5000 lines, but some are 14000 lines! 

Can we share common types between modules?